src/panels/config/automation/trigger/types/ha-automation-trigger-webhook.ts

```diff
@@ -1,24 +1,83 @@
 import "@polymer/paper-input/paper-input";
-import { html, LitElement } from "lit";
-import { customElement, property } from "lit/decorators";
-import { WebhookTrigger } from "../../../../../data/automation";
+import "../../../../../components/ha-icon-button";
+import { UnsubscribeFunc } from "home-assistant-js-websocket";
+import { mdiContentCopy } from "@mdi/js";
+import { css, html, LitElement } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import type { PaperInputElement } from "@polymer/paper-input/paper-input";
+import { fireEvent } from "../../../../../common/dom/fire_event";
+import { slugify } from "../../../../../common/string/slugify";
+import { copyToClipboard } from "../../../../../common/util/copy-clipboard";
+import { showToast } from "../../../../../util/toast";
+import {
+  WebhookTrigger,
+  AutomationConfig,
+} from "../../../../../data/automation";
 import { HomeAssistant } from "../../../../../types";
 import { handleChangeEvent } from "../ha-automation-trigger-row";
 
+const DEFAULT_WEBHOOK_ID = "default webhook id";
```

Comment:
Why not make this an empty string ?

Reply:
It is now. The (incorrect) way I was doing it before in `render` caused clearing the textfield contents to regenerate the ID. Probably not what the user wanted. Your suggestion to use `willUpdate` made this a lot simpler. Thanks!